paddle/fluid/framework/executor.cc

```diff
@@ -351,8 +372,22 @@ void Executor::RunPreparedContext(ExecutorPrepareContext* ctx, Scope* scope,
 #endif
   }
 
+  if (FLAGS_use_ngraph) {
+#ifdef PADDLE_WITH_NGRAPH
+    fused_ops_.clear();
+#endif
+  }
+
   for (auto& op : ctx->ops_) {
-    op->Run(*local_scope, place_);
+    VLOG(3) << place_ << " " << op->DebugStringEx(local_scope);
+
+    if (FLAGS_use_ngraph) {
+#ifdef PADDLE_WITH_NGRAPH
+      fused_ops_.push_back(std::move(op));
```

Comment:
Let's figure out a cleaner way to do it.
Perhaps it can be done in Prepare phase. Or in a pass

Reply:
> Or in a pass

I agree with it.

Reply:
Will look into this

Reply:
We can
1. Have FusedOperator inherit OperatorBase
2. Fusing Paddle operators into FusedOperator is done in PrepareXXX() method and encapsulated into a function.

So no change would be needed in Run()

Reply:
done